x/incentives/keeper/distribute_test.go

```diff
@@ -334,73 +376,51 @@ func (s *KeeperTestSuite) TestDistribute_InternalIncentives_NoLock() {
 				gauges = append(gauges, *gauge)
 			}
 
-			// Distribute tokens from the gauge
+			// System under test: Distribute tokens from the gauge
 			totalDistributedCoins, err := s.App.IncentivesKeeper.Distribute(s.Ctx, gauges)
-			if tc.expectErr {
-				s.Require().Error(err)
-
-				// module account amount must stay the same
-				balance := s.App.BankKeeper.GetAllBalances(s.Ctx, s.App.AccountKeeper.GetModuleAddress(types.ModuleName))
-				s.Require().Equal(coinsToMint, balance)
-
-				for i, gauge := range gauges {
-					for j := range gauge.Coins {
-						incentiveId := i*len(gauge.Coins) + j + 1
-
-						// get poolId from GaugeId
-						poolId, err := s.App.PoolIncentivesKeeper.GetPoolIdFromGaugeId(s.Ctx, gauge.GetId(), currentEpoch.Duration)
-						s.Require().NoError(err)
-
-						// check that incentive record wasn't created
-						_, err = s.App.ConcentratedLiquidityKeeper.GetIncentiveRecord(s.Ctx, poolId, currentEpoch.Duration, uint64(incentiveId))
-						s.Require().Error(err)
-					}
-				}
-			} else {
```

Comment:
It seems a messed up diff is unavoidable here. The changes made are:
1. Removed error branch, as the logic covered by these cases should never be erroring (also there were no error cases)
2. Moved the passing logic out of the `else` block, which caused the indentation-related diff below

Reply:
This is fine to me imo